Build issues with Solo5

Best effort at integrating checkseum with a Solo5 unikernel here:
```
git clone https://github.com/mato/camel-service
cd camel-service/checkseum-unikernel
mirage configure --target hvt
make depend
make
mirage build
ld: cannot find -l:freestanding/liblaolao_freestanding_stubs.a
run ['ld' '-nostdlib' '-z' 'max-page-size=0x1000' '-static' '-T'
     '/home/g2p/.opam/4.07.0+fp+flambda/lib/pkgconfig/../../lib/solo5-bindings-hvt/solo5_hvt.lds'
     '/home/g2p/.opam/4.07.0+fp+flambda/lib/pkgconfig/../../lib/solo5-bindings-hvt/solo5_hvt.o'
     '_build/main.native.o'
     '-L/home/g2p/.opam/4.07.0+fp+flambda/lib/checkseum'
     '-l:freestanding/liblaolao_freestanding_stubs.a'
     '-L/home/g2p/.opam/4.07.0+fp+flambda/lib/zarith' '-lzarith-freestanding'
     '-L/home/g2p/.opam/4.07.0+fp+flambda/lib/gmp-freestanding'
     '-lgmp-freestanding' '-L/home/g2p/.opam/4.07.0+fp+flambda/lib/nocrypto'
     '-lnocrypto_stubs+mirage-freestanding'
     '-L/home/g2p/.opam/4.07.0+fp+flambda/lib/mirage-entropy'
     '-lmirage-entropy_stubs+mirage-freestanding'
     '-L/home/g2p/.opam/4.07.0+fp+flambda/lib/pkgconfig/../../lib/ocaml-freestanding'
     '/home/g2p/.opam/4.07.0+fp+flambda/share/pkgconfig/../../lib/mirage-solo5/libmirage-solo5_bindings.a'
     '-lasmrun' '-lnolibc' '-lopenlibm' '-o' 'http.hvt']: exited with 1
make: *** [Makefile:18: build] Error 1
```

And a similar error with the checkseum.ocaml variant.

Removing the last line from the META file fixes the build for the OCaml variant.

--- a/META.checkseum.template
+++ b/META.checkseum.template
@@ -1,4 +1,4 @@
 # DUNE_GEN
 
-xen_linkopts = "-l:xen/liblaolao_xen_stubs.a"
-freestanding_linkopts = "-l:freestanding/liblaolao_freestanding_stubs.a"
+xen_linkopts = "-l:laolao/xen/liblaolao_xen_stubs.a"
+freestanding_linkopts = "-l:laolao/freestanding/liblaolao_freestanding_stubs.a"

